src/shared/ui/main-title/main-title.tsx

```diff
@@ -0,0 +1,21 @@
+import { VariantProps, cva } from 'class-variance-authority';
+import classNames from 'classnames/bind';
+
+import styles from './main-title.module.scss';
+
+type MainTitleProps = React.HTMLAttributes<HTMLHeadingElement> & VariantProps<typeof mainTitleVariants>;
+
+export const cx = classNames.bind(styles);
+
+export const mainTitleVariants = cva(cx('title'));
+
+export const MainTitle = ({ children, className, ...props }: MainTitleProps) => {
```

Comment:
I don't like that we are writing props differently. We need to decide the right way

![image](https://github.com/rolling-scopes/site/assets/114234698/7ad0f41d-c057-4543-a693-1ba380200993)

Reply:
Let's stick with an approach that doesn't use `React.FC` https://github.com/facebook/create-react-app/pull/8177